src/item.cpp

```diff
@@ -12652,7 +12652,7 @@ bool item::process_internal( Character *carrier, const tripoint &pos,
             return true;
         }
     } else {
-        // guns are never active so we only need thck this on inactive items. For performance reasons.
+        // guns are never active so we only need thick this on inactive items. For performance reasons.
```

Comment:
Shouldn't it be "tick"?

Reply:
yeah, it should be tick, I can't believe I missed that